configs/experiment_schema.json

```diff
@@ -0,0 +1,118 @@
+{
+  "title": "Experiment schema",
+  "type": "object",
+  "minProperties": 1,
+  "additionalProperties": false,
+  "properties": {
+    "rounds":{
+      "type": "array",
+      "minItems": 1,
+      "items":{
+        "$ref": "#/definitions/round"
+      }
+    }
+  },
+  "definitions": {
+    "round": {
+      "type": "array",
+      "minItems": 1,
+      "items":{
+        "$ref": "#/definitions/prefixAnnouncement"
+      }
+    },
+    "prefix": {
+      "type": "string",
+      "pattern": "^[0-9.]+/[0-9]+$"
+    },
+    "ASN": {
+      "type": "number",
+      "minimum": 1,
+      "maximum": 4294967295
+    },
+    "allocatedPrefix": {
+      "$ref": "#/definitions/prefix",
+      "enum": ["184.164.224.0/24"]
+    },
+    "peeringASN": {
+      "$ref": "#/definitions/ASN",
+      "enum": [47065, 33207, 61574, 61575, 61576, 263842, 263843, 263844]
+    },
+    "peeringMux": {
+      "type": "string",
+      "enum": [
+        "amsterdam01",
+        "clemson01",
+        "gatech01",
+        "grnet01",
+        "isi01",
+        "neu01",
+        "phoenix01",
+        "seattle01",
+        "ufmg01",
+        "utah01",
+        "uw01",
+        "wisc01",
+        "mux1",
+        "mux2"
+      ]
+    },
+    "announcement": {
+      "type": "object",
+      "required": ["muxes"],
+      "properties": {
+        "muxes": {
+          "type": "array",
+          "minItems": 1,
+          "uniqueItems": true,
+          "items": {
+            "$ref": "#/definitions/peeringMux"
+          }
+        },
+        "peers": {
+          "type": "array",
+          "uniqueItems": true,
+          "items": {
+            "type": "number"
+          }
+        },
+        "prepend": {
```

Comment:
We should discuss whether we want to include BGP poisoning here, and disallow it in the API.